src/test/run-make-fulldeps/hotplug_codegen_backend/the_backend.rs

```diff
@@ -20,7 +20,7 @@ use rustc::ty::query::Providers;
 use rustc::middle::cstore::{EncodedMetadata, MetadataLoader, MetadataLoaderDyn};
 use rustc::dep_graph::DepGraph;
 use rustc::util::common::ErrorReported;
-use rustc_codegen_utils::codegen_backend::CodegenBackend;
+use rustc_codegen_ssa::traits::CodegenBackend;
```

Comment:
You need to replace the `extern crate rustc_codegen_utils;` above with `extern crate rustc_codegen_ssa;`. Rustc private crates are not in the implicit prelude.

Reply:
Thank you!